protobuf/github.com/tendermint/tendermint/abci/types/types.proto

```diff
@@ -226,10 +228,10 @@ message BlockParams {
   int64 max_gas = 2;
 }
 
-// EvidenceParams contains limits on the evidence.
 message EvidenceParams {
   // Note: must be greater than 0
-  int64 max_age = 1;
+  int64 max_age_num_blocks = 1;
+  google.protobuf.Duration max_age_duration = 2 [(gogoproto.nullable)=false, (gogoproto.stdduration)=true];
```

Comment:
I'm a bit puzzled why this diff shows up when it's like that on develop: https://github.com/tendermint/rust-abci/blob/develop/protobuf/abci/types/types.proto#L227

can you perhaps rebase on the latest develop?